### DISCUSSION TITLE
[Alpha] Process

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2583] [Komponenter]
## Velkommen til vurdering av Process 🙌
`Process` brukes til å vise og forklare en serie med hendelser/opplysninger. Hvert steg kan være til informasjon, ha handlinger, lenke til andre sider eller fortelle status med `Tags`.

Forslaget her er å kommunisere hendelser som skjer etter hverandre på en fleksibel måte. Det skal støtte flere tidsperspektiver (fortid, nåtid og fremtid). Process kan erstatte komponenten [Stepper](https://aksel.nav.no/komponenter/core/stepper). 
<img width="1964" alt="Demo Process" src="https://github.com/navikt/aksel/assets/82579704/f77ac9d8-0ae4-426f-85be-b8347e115bf1">

Vi har lagt et eksempel på komponenten [Process i Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=1634-28391&mode=design) som dere kan teste ut.

For å finne ut om vi skal inkludere `Process` som en komponent i Aksel trenger vi deres hjelp. Vi er på jakt etter bevis på behov i løsninger som er i prod/demo. Vi ønsker skjermbilder (uten sensitive opplysninger) og en forklaring av konteksten.

Vi holder ikke aktivt øye med denne diskusjonen, så gi oss et hint om du legger til noe her. Når vi har fått bevis på minst 3 tilfeller av bruk vil vi gå videre med komponenten. Del dette gjerne med team du tror `Process` vil passe for.

**Comment by cskrov** [comment]
Vi i Team Klage hadde hatt nytte av denne komponenten nå, for å forklare klage- og ankeprosessen til brukere.
I tillegg til å forklare prosessen vil vi også lenke til relevante dokumenter og muligheter for tidligere og fremtidige steg, f.eks. ettersendelse.

_Det ser ut som alle stegene i skissene har dato.
For fremtidige muligheter bruker har, kommer vi til å trenge steg uten dato. F.eks. anke og Trygderetten._

Planen er i første omgang å bruke denne på Min Side, men den vil også kunne være nyttig for vår applikasjon for innsending av klager, anker og ettersendelser.

Har ingen skjermbilder å vise, men kommer til å implementere denne komponenten nå.

**Comment by martineeklund** [comment]
Denne ville vært nyttig på innloggede sider for å forklare hjelpemiddelbrukerne gangen i hjelpemiddelsaker, sånn at de lettere kan ha oversikt over egen sak. Dessverre kan vi ikke prioritere å forbedre disse sidene nå, så vi kommer ikke til å ta i bruk komponenten ennå, vil bare melde at vi ser behovet.

**Comment by ole-kristian-rudjord** [comment]
Jeg jobber i team BAKS (Barnetrygd og Kontantstøtte). Vi utvikler en oversikt som viser hvor langt en brukers søknad har kommet i søknadsprosessen. Til nå har vi brukt `Stepper` komponenten som eksempel, der vi har lagt til komponenter i `children` prop'en, selv om den egentlig forventer en `string`. Dette gjør at vi ikke kan bruke `Stepper` slik den er i dag. Hvis `Process` komponenten blir laget, vil vi kunne ta den i bruk i stedet.

Slik ser eksempelet vårt ut når vi brukte `Stepper`:

![image](https://github.com/user-attachments/assets/37004167-ddfb-42b3-8a23-23c052113dbc)

**Comment by sjur-gr** [comment]
[Process er lansert 🎉](https://aksel.nav.no/grunnleggende/endringslogg/2025-08-ny-komponent-process)
